src/beautifiers/hindent.coffee

```diff
@@ -0,0 +1,27 @@
+###
+Requires https://github.com/commercialhaskell/hindent
+###
+
+"use strict"
+Beautifier = require('./beautifier')
+
+module.exports = class Hindent extends Beautifier
+  name: "hindent"
+  link: "https://github.com/commercialhaskell/hindent"
+  isPreInstalled: false
+
+  options: {
+    Haskell: true
```

Comment:
`true` indicates all options are supported. Looking at code below, this should be `false` indicating language is supported with no supported options.